projects/core/src/lib/util/components/backend-status-page/backend-status-page.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="container" fxLayout="row" fxLayoutAlign="center center">
+    <mat-card class="info" *ngIf="backendInfo">
+        <mat-card-title>
+            <mat-icon mat-card-md-image svgIcon="geoengine:logo-alt" aria-label="Geo Engine" [inline]="true"></mat-icon>
+            <mat-card-title>Backend Status</mat-card-title>
+            <mat-card-subtitle>Details about the connected backend</mat-card-subtitle>
+        </mat-card-title>
+        <mat-card-content>
+            <mat-list-item>
+                <h3 matLine>backend version</h3>
+                <p matLine>{{ backendInfo.version }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>backend features</h3>
+                <p matLine>{{ backendInfo.features }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>build date</h3>
+                <p matLine>{{ backendInfo.buildDate }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>commmit hash</h3>
+                <p matLine>{{ backendInfo.commitHash }}</p>
+            </mat-list-item>
+        </mat-card-content>
+
+        <mat-card-actions>
+            <button mat-button color="primary" (click)="goBack()">go back</button>
+            <button mat-button (click)="refresh()">refresh status</button>
+        </mat-card-actions>
+    </mat-card>
+
+    <mat-card *ngIf="error">
+        <mat-card-title>
+            <mat-icon mat-card-md-image svgIcon="geoengine:logo-alt" aria-label="Geo Engine" [inline]="true"></mat-icon>
+            <mat-card-title>Backend Error</mat-card-title>
+            <mat-card-subtitle>The backend is not available!</mat-card-subtitle>
+        </mat-card-title>
+
+        <mat-card-content>
+            <mat-list-item>
+                <h3 matLine>Error status code</h3>
+                <p class="error" matLine>{{ error.status }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error status</h3>
+                <p class="error" matLine>{{ error.statusText }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error name</h3>
+                <p class="error" matLine>{{ error.name }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error message</h3>
+                <p class="error" matLine>{{ error.message }}</p>
+            </mat-list-item>
+        </mat-card-content>
+
+        <mat-card-actions>
+            <button mat-button color="primary" (click)="goBack()">reload UI</button>
+            <button mat-button (click)="refresh()">refresh status</button>
+        </mat-card-actions>
```

Comment:
Please use `mat-raised-button` to make them more visible.

Reply:
[f4c0c14](https://github.com/geo-engine/geoengine-ui/pull/512/commits/f4c0c14d3e2e31abf0c15b0184c865341242bdd3)